chart.py

```diff
@@ -6,17 +6,10 @@
 
 
 class Chart:
```

Comment:
This was added specifically for another Pull Request to add a second chart parser, so its needed for typing.